en_US/deploy/cluster/db.md

```diff
@@ -31,45 +31,46 @@ At the same time, they have a full local copy of the data, so the read access is
 
 This approach solves two problems:
 
-- Horizontal scalability (we’ve tested EMQX cluster with 23 nodes)
+- Horizontal scalability (we've tested EMQX cluster with 23 nodes)
 
 - It enables autoscaling of the replicant cluster
 
-Since replicant nodes don't participate in writes, efficiency of table updates doesn't suffer when more replicants are added to the cluster.
-This allows to create larger EMQX clusters.
+Since replicant nodes don't participate in writes, the efficiency of table updates doesn't suffer when more replicants are added to the cluster.
+This allows the creation of larger EMQX clusters.
 
 Also, replicant nodes are designed to be ephemeral.
 Adding or removing them won't change the data redundancy, so they can be placed in an autoscaling group, thus enabling better DevOps practices.
-Note, that initial replication of the data from the core nodes is a relatively heavy operation, depending on the size of the data, so autoscaling policy must not be very aggressive.
+Note that initial replication of the data from the core nodes is a relatively heavy operation, depending on the data size, so the autoscaling policy must not be very aggressive.
 
 ## Configuration
 
-In EMQX 5.0 all nodes assume core role by default, so without any tweaks the cluster behaves like 4.*.
+In EMQX 5.0 all nodes assume a core role by default, so without tweaks, the cluster behaves like 4.*.
```

Comment:
It feels like article is not necessary here

Reply:
✅